docs/pages/blog/first-look-at-joy.md

```diff
@@ -0,0 +1,133 @@
+---
+title: First look at Joy UI 🥳
+description: A sneak peek at MUI's new starting point for your design system.
+date: 2022-06-01T00:00:00.000Z
+authors: ['danilo-leal', 'siriwatknp']
+tags: ['News', 'MUI Core']
+card: true
+---
+
+<img src="/static/blog/first-look-at-joy/card.png" style="width: 796px; margin-bottom: 24px;" alt="First look at Joy UI: a new starting point for your design system." />
+
+If you're a close follower of everything MUI, you might be aware that we've been working on a new design system product.
+If not, we're writing this piece to give you a first look at Joy－MUI's new starting point for your design system!
+
+If you ever considered going with Material UI because of the thought through developer experience, reliability, and completeness but ended up hesitating because of the design, this is great news for you. Joy UI is meant to offer you all of that with a modern look and feel, ease of customization, and lower overhead by not carrying unused Material Design specific code and styling.
+
+## Why work on it?
+
+You might intuitively associate MUI with Material UI, which despite the indeed potentially confusing names, are separate things.
+Material UI is MUI's React implementation of Google's Material Design.
+
+Over the years Material UI has established itself as the go-to library for quickly breathing life into products, mostly thanks to its design, customizability, and documentation.
+However, the components do come with the default 2018 Google look and feel that is no longer as popular as it was.
+And, as we've reconfirmed with [our latest developer survey](/blog/2021-developer-survey-results/#what-are-your-most-important-criteria-for-choosing-a-ui-library/), the design aspect of UI libraries is one of the most important ones whenever evaluating which to choose.
+
+## Why not just build a different theme?
+
+One of the main objectives with this new product is to break away from Material Design.
+A different theme would still require you and your team to have at least some sense of how Material Design is structured, and how that is expressed in Material UI. We wanted to provide an alternative to that. With Joy, we want you to be able to start with a fresh design by default.
+
+We also had many ideas for improving the experience of developing design systems, so a separate package is an opportunity to prototype and test some of these new features without running the risk of compromising the experience of the thousands of teams that use Material UI today.
+
+Material UI and Joy UI will still share many conventions and features.
+This will not only decrease the learning curve for adopting Joy, but also because we think of them both representing the ideal developer experience for developing design systems.
```

Comment:
Iterating on the above suggestion to fix the grammar.

```suggestion
We've done this not only to decrease the learning curve for adopting Joy UI, but also because we believe that the conventions we've established represent the ideal developer experience for working with design systems.
```